packages/contracts/contracts/optimistic-ethereum/OVM/bridge/tokens/Abs_L1TokenGateway.sol

```diff
@@ -114,31 +114,39 @@ abstract contract Abs_L1TokenGateway is iOVM_L1TokenGateway, OVM_CrossDomainEnab
     /**
      * @dev deposit an amount of the ERC20 to the caller's balance on L2
      * @param _amount Amount of the ERC20 to deposit
+     * @param _data Data to forward to L2.
```

Comment:
Should we be adding a note here that the data may be zeroed out or crazy, and that it is basically a scratchpad from the perspective of these contracts?

Reply:
(ditto for all the other new instances of `* @param _data Data to forward...`)

Reply:
Seems like a reasonable thing to do.